documentation/schema.json

```diff
@@ -431,6 +442,58 @@
         }
       }
     },
+    "AzureAdOptions": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "ClientId",
+        "RequiredRole"
+      ],
+      "properties": {
+        "Instance": {
+          "type": [
+            "null",
+            "string"
+          ],
+          "description": "Specifies the Azure cloud instance users are signing in from. Can be either the Azure public cloud or one of the national clouds.",
+          "format": "uri",
+          "default": "https://login.microsoftonline.com"
+        },
+        "TenantId": {
+          "type": [
+            "null",
+            "string"
+          ],
+          "description": "The tenant id of the Azure Active Directory tenant, or its tenant domain.",
+          "default": "organizations"
```

Comment:
I was under the impression that the tenantid is a guid. Does this get resolved by aad? Are there other meaningful values here?